docs/reference/configuration-reference.md

```diff
@@ -30,4 +30,5 @@ The settings are grouped by feature or purpose, for example:
 - [Task Manager](/reference/configuration-reference/task-manager-settings.md)
 - [Telemetry](/reference/configuration-reference/telemetry-settings.md)
 - [URL drilldown](/reference/configuration-reference/url-drilldown-settings.md)
-- [Product intercept](/reference/configuration-reference/product-intercept-settings.md)
\ No newline at end of file
+- [Product intercept](/reference/configuration-reference/product-intercept-settings.md)
+- [Share](/reference/configuration-reference/share-settings.md)
```

Comment:
Sorry to ask I initially missed this, is it possible to insert those 2 last items in the list to keep alphabetical ordering?